src/progress-indicator/overlay.scss

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-progress-indicator-overlay($backgroundColor: iui-color-background-1, $pointerEvent: auto) {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  z-index: 500;
```

Comment:
z-index is not enough..
In Modal we have 1000.
I think we can easily set this to really high, like 99999, cause I don think there is any element which could go on top of it...

Reply:
Toast is 1070, which I think might not be enough.

We already saw that imodeljs uses 13000 somewhere